src/devices/Common/Iot/Device/Common/GeographicPosition.cs

```diff
@@ -132,6 +132,37 @@ public static void GetDegreesMinutesSeconds(double angle, int secDigits, out dou
             }
         }
 
+        /// <summary>
+        /// Returns the given angle as degrees, minutes and seconds
```

Comment:
```suggestion
        /// Returns the given angle as degree and minutes
```

not seeing seconds anywhere in the out params